client/src/main.rs

```diff
@@ -65,7 +65,7 @@ fn main() -> std::io::Result<()> {
     )
     .map_err(|e| std::io::Error::new(std::io::ErrorKind::Other, &format!("{}", e)[..]))?;
 
-    // Test connection to validator
+    /// Test connection to validator
```

Comment:
This causes a `warning: unused doc comment` warning.

Reply:
Thanks, didn't know it :p